src/Chapter2/Isbn.hs

```diff
@@ -0,0 +1,22 @@
+module Chapter2.Isbn where
+import Data.Char
+import Data.List
+
+rmdash :: String -> String
+rmdash str = filter (/='-') str
+cformat str = if (False `elem` (map (\c -> (isDigit c)) (rmdash str))) || (length (rmdash str) /= 10)
+    then False
```

Comment:
having an `if-then-else` with the `then` and `else` evaluate to `Bool` doesn't make sense.